### PR TITLE
Expand Search Arc suggestions to available engines

### PR DESCRIPTION
**Pull request by hyugin**
## Description

Update the "Search Arc" suggestions to allow a user to select their preferred search engine

I am open to any feedback or criticism, as some of the search engines don't seem to have their suggestion api/url public or gated between an authentication layer

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @hugivar 👋 

Do you mind adding a changelog entry too

@thomaspaulmann or @zach-fuller do you mind checking this?

File: extensions/arc/package.json [hunk 1]
```diff
@@ -110,6 +111,40 @@
...
+              "title": "Bing",
+              "value": "bing"
+            },
+            {
+              "title": "Yahoo!",
+              "value": "yahoo"
+            },
+            {
+              "title": "Neeva",
+              "value": "neeva"
```
Comment (thomaspaulmann) [inline extensions/arc/package.json:140]: Neeva closes their search engine business. Lets remove it from these options.

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
This is a great addition and many people will approach this. I left a small comment to remove Neeva given the recent news that they stop their user-facing search engine.

File: extensions/arc/package.json (hunk 1 above)
Comment (hugivar) [reply extensions/arc/package.json]: @thomaspaulmann thanks for the suggestion. Updated!

File: extensions/arc/package.json [hunk 2]
```diff
@@ -110,6 +111,36 @@
       "description": "Search tabs, history and search suggestions",
       "mode": "view",
       "preferences": [
+        {
+          "name": "engine",
+          "title": "Search Engine",
+          "description": "SSearch engine used to find search results",
```
Comment (thomaspaulmann) [inline extensions/arc/package.json:117]:
```suggestion
          "description": "Search engine used for suggestions and ",
```

**Comment by pernielsentikaer** [comment]
Hi @hugivar 👋 

I added some new stuff to this PR, do you mind checking what I changed, I hope nothing breaks 🙂

There might be scenarios that I didn't think about, so I'd appreciate any help 🙂

**Comment by hugivar** [comment]
@pernielsentikaer, 

The code looks good to me. I've built this locally and let you know if I see anything

**Pull request merged**